izer/add_fake_passthrough.py

```diff
@@ -61,10 +61,10 @@ def main():
         new_state_dict[name] = v
 
     new_state_dict[f'{args.layer_name}.output_shift'] = torch.Tensor([1.]).to(device)
-    new_state_dict[f'{args.layer_name}.weight_bits'] = torch.Tensor([8.]).to(device)
+    new_state_dict[f'{args.layer_name}.weight_bits'] = torch.Tensor([2.]).to(device)
     new_state_dict[f'{args.layer_name}.bias_bits'] = torch.Tensor([8.]).to(device)
     new_state_dict[f'{args.layer_name}.quantize_activation'] = torch.Tensor([1.]).to(device)
-    new_state_dict[f'{args.layer_name}.adjust_output_shift'] = torch.Tensor([0.]).to(device)
+    new_state_dict[f'{args.layer_name}.adjust_output_shift'] = torch.Tensor([1.]).to(device)
```

Comment:
output_shift should have stayed 0. With 1, it computes output_shift from the weights and does not use our set output_shift value, which is 1. Most probably, it comes up with 1 from the calculation, but we can save by not doing the quantile calculation.

Reply:
Realized in the latest commit.